src/main/java/com/suse/matcher/OptaPlanner.java

```diff
@@ -99,155 +77,17 @@ public OptaPlanner(Assignment unsolved, boolean testing) {
      * @return the solver
      * @param testing true if running as a unit test, false otherwise
      */
-    @SuppressWarnings("rawtypes")
     private Solver initSolver(boolean testing) {
-        // init basic objects
```

Comment:
I will not hide I am a little bit sad to see this code go, it really took a lot of care to make sure it was as readable and self-explaining as possible.

OTOH, I see the need for proper benchmarks in the long run and recognize that as you will likely be the one needing to work on it, it's best for you to own this bit and ultimately decide what to do.

So it goes away... But for a good cause :-)